### DISCUSSION TITLE
Do I need to change these versions in composer.json?

### DISCUSSION THREAD

**Discussion by ghost** [symfony/symfony#42642] [Q&A] [answered]
In my composer.json I have this versions:

```
    "doctrine/doctrine-bundle": "^2.4",
    "doctrine/doctrine-migrations-bundle": "^3.1",
    "doctrine/orm": "^2.9",
    "symfony/flex": "^1.3.1",
    "twig/extra-bundle": "^2.12|^3.0",
    "twig/twig": "^2.12|^3.0",

```
But they have a new version:

```
    doctrine/doctrine-bundle => 2.4.2
    doctrine/doctrine-migrations-bundle => 3.1.1
    doctrine/orm => 2.9.4
    symfony/flex => 1.13.4
    twig/extra-bundle => 3.3.1
    twig/twig => 3.3.2

```
Do I need to change these versions in composer.json?

** I use Symfony 5.3.6

**Comment by romaricdrigon** [comment, marked as answer]
`^2.9` means "higher version still compatible with 2.9", which usually means >= 2.9, but not 3.0.
So "2.9.4" should match - you likely just need to run `composer update`.

**Reply by zspine** [reply]
Yes, a simple ```composer update``` command will sufficient.

Run ```composer show --direct``` to see the installed package versions if you want more information about the installed packages.

To check for outdated packages, run the command ```composer outdated --direct```.
